### PR TITLE
chore: Don't include langtags for Sign Languages

### PR DESCRIPTION
**Pull request by darcywong00**
The recent langtags.json file is including language tags for several Sign Languages (most use `Zxxx` script).
This updates the parse-langtags utility to ignore them since Speech Analyzer handles audio files and not video files used for Sign Languages.

Additional updates:
* Change GHA from weekly to monthly
* Incorporate validation of the langtags.json file (from silnrsi/langtags#5)

### CODE REVIEW

File: DevUtils/parse-langtags/src/index.ts [hunk 1]
```diff
@@ -98,11 +110,29 @@ function loadLangtagsJson() : any {
   return langtagsJson;
 }
 
+// Validate langtags.json file
+function validateLangtagsJson(langtagsJson : any[]) : boolean {
```
Comment (kgorham) [inline DevUtils/parse-langtags/src/index.ts:114]: it seems the incoming json would only ever by strings.  suggest using string instead of 'any'

Comment (megahirt) [reply DevUtils/parse-langtags/src/index.ts:114]: I'm not 100% sure about that.

File: DevUtils/parse-langtags/langtags.schema.json [hunk 2]
```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$ref": "#/definitions/langtags",
+  "definitions": {
```
Comment (megahirt) [inline DevUtils/parse-langtags/langtags.schema.json:4]:
```suggestion
  "$comment": "See https://gist.github.com/mcdurdin/eff65bb2d77034525120d48cbe280d30"
  "definitions": {
```

Let's add a comment to the schema referring back to where it came from

**Review by megahirt — APPROVED** [review]
LGTM

One suggestion to add a reference to the schema originally published by @mcdurdin

File: DevUtils/parse-langtags/langtags.schema.json (hunk 2 above)
Comment (mcdurdin) [reply DevUtils/parse-langtags/langtags.schema.json:5]: I'd really like to see the schema defn adopted by the langtags repo @mhosken 😁

Comment (mcdurdin) [reply DevUtils/parse-langtags/langtags.schema.json:5]: See https://github.com/silnrsi/langtags/blob/master/source/langtags_schema.json now for official schema source

**Pull request merged**